### ISSUE TITLE
renamed speedtest 

### ISSUE THREAD

**Issue by kbrouder** [pschmitt/zabbix-speedtest#3]
seems like you renamed speedtest to speedtest-linux-amd64

This makes your readme very hard to follow but more importantly, theres a bug in speedtest.sh on line 23 as its referencing "speedtest".

Unless I have mine completely setup wrong, I dont think your master will build as is

**Comment by pschmitt** [comment]
Well there are 2 version of the speedtest binary, you need to choose the right one depending on your architecture.

**Comment by jpmorris** [comment]
this definitely is not clear.  if there are two versions there shoudl be a link to the correct one (I still cant find it).  nothing about renaming. this is obviously not clear.   I've never done a pull request and not sure if I have time.  I do know it's takes 10 seconds for the maintainer to change the Readme through the github website

**Comment by BearGFR** [comment]
The github version of speedtest-cli has issues that cause it to return incorrect values in some cases.  I obtained the "official" version of the speedtest command line interface from Ookla (https://www.speedtest.net/apps/cli)  and have it working like a champ on my system now.  It did require changes to the script, some additional directories/permissions to make it all work, but working it is.  I don't mind sharing, if anyone is interested.  I'll also add this comment as an issue for more visibility.
